src/main/java/software/sigma/internship/repository/LocaleRepository.java

```diff
@@ -0,0 +1,7 @@
+package software.sigma.internship.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import software.sigma.internship.entity.Locale;
+
+public interface LocaleRepository extends JpaRepository<Locale, Long> {
```

Comment:
We don't need locale repository for now as we don't plan to update them through code